plone/recipe/haproxy/__init__.py

```diff
@@ -121,7 +120,7 @@ def install(self):
                 target = os.path.join(bintarget, filename)
                 with open(target, "wt") as f:
                     f.write("#!/bin/sh\n")
-                    f.write('exec {} "$@"'.format(os.path.join(fullpath, filename)))
+                    f.write('exec {} "$@"'.format(os.path.join(fullpath, filename)))  # noqa
```

Comment:
Note that `# noqa: E501` is preferrable, see https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors